ucm2/Qualcomm/sc8280xp/HiFi.conf

```diff
@@ -0,0 +1,89 @@
+# Use case configuration for LenovoX13s.
+# Author: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
+
+SectionVerb {
+
+	EnableSequence [
+		cset "name='RX_CODEC_DMA_RX_0 Audio Mixer MultiMedia1' 1"
+		cset "name='WSA_CODEC_DMA_RX_0 Audio Mixer MultiMedia2' 1"
+		cset "name='MultiMedia4 Mixer VA_CODEC_DMA_TX_0' 1"
+		cset "name='MultiMedia3 Mixer TX_CODEC_DMA_TX_3' 1"
+	]
+
+	Include.wcde.File "/codecs/wcd938x/DefaultEnableSeq.conf"
+	Include.wsae.File "/codecs/wsa883x/DefaultEnableSeq.conf"
+
+	Include.wcdd {
+		File "/codecs/wcd938x/DefaultDisableSeq.conf"
+		Before.DisableSequence "0"
```

Comment:
Why Before ? It's not required to re-index things when only one DisableSequence is present.

Reply:
Thanks for the hints,  will remove this.